formulaic/models.py

```diff
@@ -874,6 +887,7 @@ class SubmissionKeyValue(models.Model):
 
     @property
     def value(self):
+        # print("THIS ONE AT LEAST?")
```

Comment:
But this was so useful...

Reply:
You can tell I'm getting frustrated by the Caps lock being left on.